data/svgs/README.md

```diff
@@ -82,6 +82,7 @@ as of the time of import. See the libratbag COPYING file for details.
  logitech-g500s.svg
  logitech-g502.svg
  logitech-g600.svg
+ logitech-g602.svg
```

Comment:
this list doesn't need to be updated, it's just the list of devices imported from libratbag which doesn't apply here